### PR TITLE
Added "Intro to Open Source" event to Events page

### LINKED ISSUE #238: Add "Intro to Open Source" event to Events page
## Description

As a newcomer,
I need see all the community events,
so that I can learn more about the community history and contributions to the community.

Event page: https://community.anitab.org/event/intro-to-open-source/
**Need image shared on social media** 🤔 

### Update [Required]
- [ ] Add "Intro to Open Source" event to Events page

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hours

### PR DESCRIPTION
**Pull request by The-Keshav-Agarwal**
### Description

Added "Intro to Open Source" event to Events page.

Fixes #238

### Type of Change:
- code

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/content/events_and_highlights.js [hunk 1]
```diff
@@ -13,6 +13,31 @@ const events_highlight = {
...
+            {
+              par: 'During the last 15 to 20 minutes, attendees can ask any questions about contributing to open source.'
+            }
+          ],
+          highlights:{
+              source: require('./../assets/events_and_highlights/5.jpg'),
+          },
+          know_more:
+          {
+            par: 'Feel free to contact AnitaB.org Open Source on Zulip  if you need any further information about this event.',
```
Comment (isabelcosta) [inline src/content/events_and_highlights.js:37]:
```suggestion
            par: 'Feel free to contact AnitaB.org Open Source on Zulip if you need any further information about this event.',
```

File: yarn.lock [hunk 2]
```diff
@@ -405,6 +405,15 @@
...
 
+"@babel/highlight@^7.10.4":
+  version "7.13.10"
+  resolved "https://registry.yarnpkg.com/@babel/highlight/-/highlight-7.13.10.tgz#a8b2a66148f5b27d666b15d81774347a731d52d1"
+  integrity sha512-5aPpe5XQPzflQrFwL1/QoeHkP2MsA4JCntcXHRhEsdsfPVkvPi2w7Qix4iV7t5S/oC9OodGrggd8aco1g3SZFg==
+  dependencies:
+    "@babel/helper-validator-identifier" "^7.12.11"
+    chalk "^2.0.0"
+    js-tokens "^4.0.0"
+
```
Comment (isabelcosta) [inline yarn.lock:416]: Is this needed for this task @The-Keshav-Agarwal ? or did this happen unintentional?

**Review by isabelcosta — COMMENTED** [review]
@The-Keshav-Agarwal thank you for contributing! This looks good just looking at the code :)

Could you please provide an image, in the PR description, of how this looks like when you run it locally?

File: yarn.lock (hunk 2 above)
Comment (The-Keshav-Agarwal) [reply yarn.lock:416]: @isabelcosta actually this happened unintentionally. is it ok if I remove this code manually?

**Comment by The-Keshav-Agarwal** [comment]
> @The-Keshav-Agarwal thank you for contributing! This looks good just looking at the code :)
> 
> Could you please provide an image, in the PR description, of how this looks like when you run it locally?

@isabelcosta here are the ss of changes
image
image

**Comment by The-Keshav-Agarwal** [comment]
@isabelcosta thanks for reviewing the PR. I made all the changes which you suggested can you please review it again.

**Comment by isabelcosta** [comment]
@The-Keshav-Agarwal could you edit the description of the pull request and add the image there? It makes it easier to review when other reviewers come to check this :) I can also do this for you, but wanted to let you know that you could do that, in case you don't know


image

**Review by isabelcosta — APPROVED** [review]
This looks good now @The-Keshav-Agarwal :)
Thank you for contributing and addressing the changes 💯 
Will request more reviewers 🤗

**Comment by The-Keshav-Agarwal** [comment]
> @The-Keshav-Agarwal could you edit the description of the pull request and add the image there? Probably would be good in the "How Has This Been Tested?" section.
> 
> It makes it easier to review when other reviewers come to check this :) I can also do this for you, but wanted to let you know that you could do that, in case you don't know
> 
> image

Thanks for the suggestion. I have added the screenshots in the PR description.

**Comment by The-Keshav-Agarwal** [comment]
@annabauza @vj-codes @Rahulm2310 can you please review this PR.

**Comment by vj-codes** [comment]
@The-Keshav-Agarwal it looks good to me!
@isabelcosta currently the timings for all events are in different timezones, I suggest keeping them in one timezone - UTC for better user experience and consistency. 
Since it's a small change can be done in this PR as well. What do you think about it?

**Comment by isabelcosta** [comment]
@vj-codes I think it's good either way. We could also leave that small change for new contributors on another issue only dedicated to this 🤔 and get this one reviewed and merged as is.

**Review by vj-codes — APPROVED** [review]
@The-Keshav-Agarwal thank you for your contribution 🎉

**Comment by isabelcosta** [comment]
@vj-codes could you create the issue for what you talked about, when you have time?

**Comment by isabelcosta** [comment]
The changes made in this PR were tested locally. Following are the results:

1. Code review - Done

2. All possible responses (positive and negative tests) were tested as below:

   * _Run locally and go to Events page_  
     _Screenshot/gif_:  
image

     _Expected Result_:  "Intro to Open Source" event should show up in the Events page
     _Actual Result_: "Intro to Open Source" event showed up in the Events page
 

5. Status of PR Changed to: Ready to Merge.

**Comment by isabelcosta** [comment]
@vj-codes thank you soo much for the help you are giving with reviews across the organization! 🙌🏾

**Pull request merged**